### PR TITLE
Add image format project proposal

### PR DESCRIPTION
**Pull request by caniszczyk**
Converting the proposal from the TOB to markdown for record keeping purposes.

Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

**Comment by caniszczyk** [comment]
cc: @opencontainers/tob

File: proposals/image-format/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Proposed Roadmap
+
+* March ?? v0.0.0
+  * Import Docker v2.2 format
+* April 18th v0.1.0
+  * Spec factored for top to bottom reading with three audiences in-mind:
+    * Build system creators
+    * Image registry creators
+    * Container engine creators
+    * Test cases for each of the potential users exercising build, push, and pull verbs
```
Comment (philips) [inline proposals/image-format/README.md]: we need to remove this

Comment (caniszczyk) [inline proposals/image-format/README.md]: feel free to do a proposed update to the markdown file :)

Comment (philips) [inline proposals/image-format/README.md]: @caniszczyk https://github.com/opencontainers/tob/pull/5

Pull requests on pull requests

**Pull request merged**